service/service_test.go

```diff
@@ -616,3 +616,52 @@ func TestGetToken(t *testing.T) {
 	assert.Equal(t, existingIdentification.GetSecret().GetCreatedTimestamp(), secret.GetCreatedTimestamp())
 	assert.Equal(t, existingIdentification.GetSecret().GetExpirationTimestamp(), secret.GetExpirationTimestamp())
 }
+
+func TestVerifyToken(t *testing.T) {
+	nonExistingToken := &pblib.Identification{
+		Token: "TestVerifyToken-DoesNotExist",
+	}
+
+	cases := []struct {
+		desc   string
+		req    *pbsvc.UserRequest
+		expMsg string
+	}{
+		{"test nil request object", nil,
+			"rpc error: code = InvalidArgument desc = nil request User",
+		},
+		{"test nil identity object", &pbsvc.UserRequest{Identification: nil},
+			"rpc error: code = InvalidArgument desc = nil request identification",
+		},
+		{"test non-existent token", &pbsvc.UserRequest{Identification: nonExistingToken},
+			"rpc error: code = Unauthenticated desc = no existing token were found for user",
+		},
+	}
+
+	for _, c := range cases {
+		s := Service{}
+
+		response, err := s.VerifyToken(context.TODO(), c.req)
+		assert.EqualError(t, err, c.expMsg, c.desc)
+		assert.Nil(t, response, c.desc)
+	}
+
+	newSecret, newToken, err := insertNewToken()
+	assert.Nil(t, err)
+	assert.NotNil(t, newSecret)
+	assert.NotEmpty(t, newToken)
+
```

Comment:
missing descriptions from the 3 `assert` statements above.

Reply:
i'm not really testing it and its not specific to the unit test... not sure what to put as description for these type of asserts. It's just there just in case. Is the description really necessary?